docs/how-to/ide-setup.txt

```diff
@@ -0,0 +1,112 @@
+.. _ide-setup:
+
+#####################
+ IDE Setup (PyCharm)
+#####################
+
+Model developers may develop and debug models with their preferred IDE.
+In this guide, we show how to configure PyCharm to develop and debug
+against Determined's :ref:`Native API <tutorials_native-api>`.
+
+***************
+ Prerequisites
+***************
+
+This guide assumes that you have created a `virtualenv
+<https://virtualenv.pypa.io/en/latest/>`__ or `Conda
+<https://docs.conda.io/en/latest/>`__ environment in which you have
+installed the dependencies required for local development. For example,
+to create a Python 3.7 virtualenv:
+
+.. code::
+
+   python3.7 -m venv /path/to/env/
+   source /path/to/env/bin/activate
+
+You will then want to install the ``determined`` Python package, as well
+the TensorFlow or PyTorch library versions that your code will run
+against when submitting to the cluster. For example, if you are using
+Determined's `default task container image
+<https://github.com/determined-ai/environments/blob/master/Makefile>`__
+for a PyTorch model, you would install the following dependencies in
+your virtualenv:
+
+.. code::
+
+   pip install 'determined[tensorflow]'==<Determined version>
```

Comment:
If we want to use PyTorch, why `determined[tensorflow]`?

Reply:
removed TF as it's not needed here